src/scf-release/src/acceptance-tests-brain/test-scripts/cf_usb_helper.rb

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env ruby
+
+## Explanations and overview:
+
+## This test does a lot of setup to determine that the universal
+## service broker (cf-usb) is actually working.
+#
+## 0. A domain for TCP routing is created, to connect all the pieces.
+#
+## 1. A local server is started as an app and made available
+##    through cf TCP routing.  The test-specific class determines how the server
+##    will be started.
+#
+## 2. The sidecar is started as an app, configured to talk to the server from
+##    the first step.
+#
+## 3. The cf-usb is configured to talk to and use the sidecar.
+#
+## 4. Then we can check that the appropriate service appears in the marketplace,
+##    create a service from it, and check that this service is viewable too.
+#
+## In the code below these phases are marked with "--(N)--" where N is
+## the step number.
+#
+## Note, the applications of step 1 and 2 are docker apps. This is why
+## the `pre-start.sh` script enables the `diego_docker` feature-flag
+## of CF. For step (3) the `pre-start.sh` script extended the `cf`
+## client with the `cf-usb-plugin` plugin.
+
```

Comment:
Should possibly document what methods derived classes are expected to define, i.e. the hooks the helper invokes for the engine specific actions.

Reply:
Fixed in 8c5d663aaab02b17417acd57003a6ffaf66d00ac